openfisca_web_api/loader/entities.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals, print_function, division, absolute_import
+from openfisca_core.commons import to_unicode
+
+
+def build_entities(tax_benefit_system):
+    entities = {
+        entity.key: build_entity(entity)
+        for entity in tax_benefit_system.entities
+        }
+    return entities
+
+
+def build_entity(entity):
+
+    formatted_entity = {
+        'plural': entity.plural,
+        'description': to_unicode(entity.label),
+        'documentation': to_unicode(entity.doc)
```

Comment:
`documentation` seems to often start and end with `\n`, which is not really relevant. We should probably clean these :)

Reply:
I found a way but it is not very elegant.... I'm looking forward to your feedback on it :)